pyroaring/__init__.pyi

```diff
@@ -0,0 +1,253 @@
+import array
+from typing import overload, TypedDict
+from collections.abc import Iterable, Iterator
+
+from typing_extensions import Self
+
+__version__: str
+__croaring_version__: str
+
+
+class _Statistics(TypedDict):
+    # Type as observed in the returned values.
+    # Warning: This type does not exist at runtime.
+
+    n_containers: int
+    n_array_containers: int
+    n_run_containers: int
+    n_bitset_containers: int
+    n_values_array_containers: int
+    n_values_run_containers: int
+    n_values_bitset_containers: int
+    n_bytes_array_containers: int
+    n_bytes_run_containers: int
+    n_bytes_bitset_containers: int
+    max_value: int
+    min_value: int
+    sum_value: int
+    cardinality: int
```

Comment:
That made me think, it could be nicer to replace this dict by a proper dataclass. But this is out of scope of this PR of course.